src/library_fs.js

```diff
@@ -1271,13 +1271,13 @@ mergeInto(LibraryManager.library, {
       FS.mkdev('/dev/tty1', FS.makedev(6, 0));
       // setup /dev/[u]random
       var random_device;
-      if (typeof crypto !== 'undefined') {
+      if (typeof crypto !== 'undefined' && crypto['getRandomValues'] !== 'function') {
```

Comment:
this check should be `typeof crypto === 'object' && crypto['getRandomValues'] !== 'function'

Reply:
I just realized it should be `typeof crypto === 'object' && typeof crypto['getRandomValues'] === 'function'`
Otherwise we would go in this branch in the wrong case

Reply:
Oh right, the condition was reversed.